### DISCUSSION TITLE
Registration + verification still functional?

### DISCUSSION THREAD

**Discussion by jpstotz** [AsamK/signal-cli#1616] [Q&A] [answered]
I just tried to register a new account using my landline number. The description how to use CAPTCHA and then switch to voice verification worked without problems. I got the verification code and then tried to verify it using `signal-cli.bat -vv -a +496151xxxxxxx verify 123456`

The response was:

`org.whispersystems.signalservice.api.push.exceptions.InvalidTransportModeException: StatusCode: 400`

I tried this two times with the same outcome. Is this a known temporary problem or had Signal changed the registration process?
or are there other obstacles that can prevent the verification?

* Client OS: Windows 10
* OpenJDK 64-Bit Server VM Temurin-21.0.5+11

All actions:
* `signal-cli.bat -a +496151xxxxxx register --captcha signalcaptcha://signal-hcaptcha-short.5fad97ac-7d06-4e44-b18a-b950b2034567.registration.A56F9afCfNS0V8_Ei1xt3Kz9`
* wait 1 minute
* `signal-cli.bat -a +496151xxxxxx register --captcha signalcaptcha://signal-hcaptcha-short.5fad97ac-7d06-4e44-b18a-b950b2034567.registration.A56F9afCfNS0V8_Ei1xt3Kz9 --voice`
* receive verification code via phone call
* `signal-cli.bat -a +496151xxxxxx verify 123456`

phone number, captcha and verify code has been modified for security/privacy reasons.

**Comment by AsamK** [comment, marked as answer]
See https://github.com/AsamK/signal-cli/issues/1614
The fix will be released to soon

**Comment by jpstotz** [comment]
Thanks, fixed with 0.13.9
